playground/components/header.tsx

```diff
@@ -32,6 +37,14 @@ export const Header: ParentComponent<{ fork?: () => void; share: () => Promise<s
     setShowMenu(false);
   }
 
+  onMount(() => {
+    props.compiler.addEventListener('message', ({ data }) => {
+      const { event, imports } = data;
+
+      if (event === 'IMPORTS') exportToZip(context.tabs()!, imports)
```

Comment:
no need to listen to errors here, as it happens already in the REPL, and we would have a double `console.error`